ui/page/send/page.go

```diff
@@ -490,7 +530,7 @@ func (pg *Page) HandleUserInteractions() {
 		go pg.fetchExchangeRate()
 	}
 
-	if pg.toCoinSelection.Clicked() {
+	if pg.toCoinSelection.Clicked() && !pg.isModalLayout { // modal layout cannot render the coinSeletion page
```

Comment:
```suggestion
	if pg.toCoinSelection.Clicked() && !pg.isModalLayout { // modal layout cannot render the coinSelection page
```

Reply:
Actually, hide this button in modal view, or disable it. If you do that, then the only time that `pg.toCoinSelection.Clicked()` will be true is if it's not modal.